packages/dds/shared-object-base/src/test/attachingBindingAndConnecting.spec.ts

```diff
@@ -5,502 +5,465 @@
-			telemetryContext?: ITelemetryContext | undefined,
-			incrementalSummaryContext?: IExperimentalIncrementalSummaryContext | undefined,
-		) => ISummaryTreeWithStats;
-		loadCore: (this: SharedObject, services: IChannelStorageService) => Promise<void>;
-		processCore: (
-			this: SharedObject,
-
-			message: ISequencedDocumentMessage,
-			local: boolean,
-			localOpMetadata: unknown,
-		) => void;
-		onDisconnect: (this: SharedObject) => void;
-		applyStashedOp: (this: SharedObject, content: any) => unknown;
-		didAttach: () => void;
-	}>,
-) {
-	class TestSharedObject extends SharedObject {
-		protected summarizeCore = overrides?.summarizeCore?.bind(this);
-		protected loadCore = overrides?.loadCore?.bind(this);
-		protected processCore = overrides?.processCore?.bind(this);
-		protected onDisconnect = overrides?.onDisconnect?.bind(this);
-		protected applyStashedOp = overrides?.applyStashedOp?.bind(this);
-		protected didAttach =
-			overrides.didAttach?.bind(this) ?? (() => assert.fail("didAttach not set"));
-	}
-
-	const runtime = overrides?.runtime ?? {};
-	runtime.channelsRoutingContext ??= {
-		absolutePath: "/",
-		attachGraph: () => {},
-		get IFluidHandleContext() {
-			return this;
-		},
-		isAttached: false,
-		resolveHandle: async () => ({ status: 500, mimeType: "error", value: "error" }),
-	};
-	runtime.IFluidHandleContext ??= runtime.channelsRoutingContext;
-
-	runtime.logger ??= createChildLogger();
-
-	const attributes = overrides?.attributes ?? {};
-
-	attributes.type ??= "TestSharedObject";
-
-	return {
-		overrides,
-		sharedObject: new TestSharedObject(
-			overrides?.id ?? "testSharedObject",
-			createOverridableProxy<IFluidDataStoreRuntime>(
-				"runtime",
-				runtime,
-				new TypedEventEmitter<IFluidDataStoreRuntimeEvents>() as any as Overridable<IFluidDataStoreRuntime>,
-			),
-			createOverridableProxy<IChannelAttributes>("attributes", attributes),
-			overrides?.telemetryConfigPrefix ?? "testSharedObject",
-		),
-	};
-}
-
-/**
- * The attach state of a shared object is determined by two facts:
- *
- * * Is it bound to a context which means its handle is stored in an already bound dds
- * * Is the runtime attached
- *
- * If both of these are true, then the dds should be considered attached.
- *
- * Beyond attach, there is also the connected state, and the didAttach method.
- * The connected state should always be false while detached, and didAttach should only be called once when the dds transitions.
- *
- * These tests valid these properties across all the functions that can cause attach state transitions
- */
-describe("SharedObject attaching binding and connecting", () => {
-	const runtimeAttachStateAndConnectedMatrix = generatePairwiseOptions({
-		connected: [true, false],
-		attachState: [AttachState.Detached, AttachState.Attaching, AttachState.Attached],
+import { Ink } from "../ink";
+import { InkFactory } from "../inkFactory";
+import { IPen } from "../interfaces";
+
+describe("Ink", () => {
```

Comment:
Something strange happened in your last commit, several test files turned into ink tests

Reply:
there is a prettier + vs code that corrupt documents when find replace is used. its very annoying :(